### ISSUE TITLE
CloudWatch Logs Tags Property Not Failing for GovCloud

### ISSUE THREAD

**Issue by taylorbartley** [aws-cloudformation/cfn-lint#2396]
### CloudFormation Lint Version

0.66.0

### What operating system are you using?

Debian GNU/Linux 11 (bullseye)

### Describe the bug

CloudWatch Logs allow tag definitions via CloudFormation, but tags are not supported in GovCloud CFN (at least not US regions).

If my .cfnlintrc.yml only Gov regions, linting fails with "E3002 Invalid Property Resources/LogGroupFoo/Properties/Tags".

If my .cfnlintrc.yml contains a commercial region (or no regions), passes when tags are added to a Log Group.

### Expected behavior

Linting should fail when a Gov region is specified in .cfnlintrc.

### Reproduction template

```yaml
AWSTemplateFormatVersion: 2010-09-09

Resources:
  LogGroupFoo:
    DeletionPolicy: Delete
    UpdateReplacePolicy: Delete
    Type: AWS::Logs::LogGroup
    Properties:
      LogGroupName: !Sub ${AWS::StackName}/foo
      RetentionInDays: 90
      Tags:
        - Key: Name
          Value: Sometag
    Metadata: # Suppressing KMS Error
      cfn_nag:
        rules_to_suppress:
          - id: W84
```

**Comment by PatMyron** [comment]
To clarify current behavior, I believe most rules are only run in the first region requested and only some rules are run for every region, which results in:
```sh
$ cfn-lint t.yml -r us-gov-west-1
E3002 Invalid Property Resources/LogGroupFoo/Properties/Tags
t.yml:11:7

$ cfn-lint t.yml -r us-gov-west-1 us-east-1
E3002 Invalid Property Resources/LogGroupFoo/Properties/Tags
t.yml:11:7

$ cfn-lint t.yml -r us-east-1 us-gov-west-1
$ cfn-lint t.yml -r us-east-1
```

This issue is mostly tracked in https://github.com/aws-cloudformation/cfn-lint/issues/607, but I also have a relevant comment in https://github.com/aws-cloudformation/cfn-lint/issues/1065#issuecomment-682276237 where the fact that we're able to catch this property-level regional disparity at all is slightly above current maintenance expectations in my opinion. Resource-level regional disparity is maintained pretty well in `cfn-lint`, but property-level regional disparities are less accurately maintained in the [CloudFormation Resource Specifications](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html)

**Comment by kddejong** [comment]
Looks like the specs now include tags for this resource type.  I can't seem to see a corresponding release notes for this.  Do you know if tags is now supported in the gov regions?

**Comment by taylorbartley** [comment]
My team actually discovered that last week. I noticed EventBridge Buses now
support cfn tagging, and followed up on CWL in Gov. Turns out, yeah there
are no issues now. I also couldn't find any release note on it.

Thanks for following up, I'd forgotten about this request.

**Comment by kddejong** [comment]
This specific issue is resolved but the underlying issue will be resolved in v1.  I'm going to close this one.  

In v1 we will validate each region if the schema has changed based on the region.
